source/events/events.rst

```diff
@@ -0,0 +1,290 @@
+* ``INSTRUME`` type: string
+    * Instrument used to aquire the data contained in the file
+      (e.g. 'North', 'South')
+* ``OBSERVER`` type: string
+    * Name of observer (e.g. 'Joe Public'). This could be for example the PI
+      of a proposal.
+* ``OBJECT`` type: string
+    * Observed object (e.g. 'Crab')
+* ``OBS_MODE`` type: string
+    * Observation mode (e.g. 'WOBBLE', 'SCAN', 'SLEW', or any mode that is
+      supported by ``TELESCOP``; string should be upper case)
+* ``OBS_ID`` type: int
+    * Unique observation identifier (Run number)
+* ``DATE-OBS`` type: string
+    * Start date of observation in ISO standard date representation
+      "ccyy-mm-ddThh:mm:ss" (UTC)
+* ``DATE-END`` type: string
+    * End date of observation in ISO standard date representation
+      "ccyy-mm-ddThh:mm:ss" (UTC)
+* ``TSTART`` type: float, unit: s
+    * Start time of observation
+      (given in instrument specific time reference, see below)
+* ``TSTOP`` type: float, unit: s
+    * End time of observation
+      (given in instrument specific time reference, see below)
+* ``MJDREFI`` type: int, unit: days
+    * Integer part of instrument specific MJD time reference
+* ``MJDREFF`` type: float, unit: days
+    * Float part of instrument specific MJD time reference       
+* ``TIMEUNIT`` type: string
+    * Time unit (e.g. 's')
+* ``TIMESYS`` type: string
+    * Time system (e.g. 'TT', 'MJD', 'JD', 'TJD')
+* ``TIMEREF`` type: string
+    * Time reference frame, used for example for barycentric corrections
+      (options: 'LOCAL', 'SOLARSYSTEM', 'HELIOCENTRIC', 'GEOCENTRIC')
+* ``EQUINOX`` type: float
+    * Equinox in years for the celestial coordinate system in which positions
+      given in either the header or data are expressed (options: 2000.0).
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
+* ``RADECSYS`` type: string
+    * Stellar reference frame used for the celestial coordinate system in
+      which positions given in either the header or data are expressed.
+      (options: 'ICRS', 'FK5').
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
+* ``TELAPSE`` type: float, unit: s
+    * Time interval between start and stop time (``TELAPSE=TSTOP-TSTART``).
+      Any gaps due to bad weather, or high background counts and/or other
+      anomalies, are included.
+* ``ONTIME`` type: float, unit: s
+    * Total *good time* (sum of length of all Good Time Intervals).
+      If a Good Time Interval (GTI) table is provided, ``ONTIME`` should be
+      calculated as the sum of those intervals. Corrections for instrumental
+      *dead time* effects are **NOT** included.
+* ``LIVETIME`` type: float, unit: s
+    * Total time (in seconds) on source, corrected for the *total* instrumental
+      dead time effect.
+* ``DEADC`` type: float
+    * Dead time correction, defined by ``LIVETIME/ONTIME``.
+      Is comprised in [0,1]. Defined to be 0 if ``ONTIME=0``.
+* ``EVENT_CLASS`` type: int
```

Comment:
I guess I took this from Fermi where it's an int (I think it's also a column there), but since we want to have these anyways in a separate file I agree that it should be "str"

> Le 3 juin 2016 à 20:49, Christoph Deil <notifications@github.com> a écrit :
> 
> In source/events/events.rst <https://github.com/open-gamma-ray-astro/gamma-astro-data-formats/pull/39#discussion_r65757700>:
> 
> > +* ``TELAPSE`` type: float, unit: s
> > +    * Time interval between start and stop time (``TELAPSE=TSTOP-TSTART``).
> > +      Any gaps due to bad weather, or high background counts and/or other
> > +      anomalies, are included.
> > +* ``ONTIME`` type: float, unit: s
> > +    * Total *good time* (sum of length of all Good Time Intervals).
> > +      If a Good Time Interval (GTI) table is provided, ``ONTIME`` should be
> > +      calculated as the sum of those intervals. Corrections for instrumental
> > +      *dead time* effects are **NOT** included.
> > +* ``LIVETIME`` type: float, unit: s
> > +    * Total time (in seconds) on source, corrected for the *total* instrumental
> > +      dead time effect.
> > +* ``DEADC`` type: float
> > +    * Dead time correction, defined by ``LIVETIME/ONTIME``.
> > +      Is comprised in [0,1]. Defined to be 0 if ``ONTIME=0``.
> > +* ``EVENT_CLASS`` type: int
> EVENT_CLASS should be type: str, no? (the examples you give are strings)
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/open-gamma-ray-astro/gamma-astro-data-formats/pull/39/files/00e584c0549f36cc18eaf9c8b87aa0f7c2c0c178#r65757700>, or mute the thread <https://github.com/notifications/unsubscribe/AC2oV6HT5tm3MIjaVYxYy7L9HAuqkZxCks5qIHdFgaJpZM4IPcoM>.
>